app/views/follow_requests/create.js.erb

```diff
@@ -0,0 +1,5 @@
+console.log("Followed")
```

Comment:
Now that your implementation is finished, you can delete all the `console.log()` statements and empty lines etc.